packages/phrases/src/locales/en.ts

```diff
@@ -7,7 +7,7 @@ const translation = {
     retry: 'Try again',
     done: 'Done',
     search: 'Search',
-    save_changes: 'Save changes',
+    save_changes: 'Save Changes',
```

Comment:
`Save changes` is also reasonable and align with other translations, e.g. `Check out`, `Set up`, `Try again`. Seems that there is no need to change it?

Reply:
Discuss in the product channel.
https://silverhand-io.slack.com/archives/C0243P3E4KS/p1655783463109529